runit/Dockerfile

```diff
@@ -18,8 +18,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
     ruby2.0 \
     zip && \
   DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends collectd-core && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/*
+    apt-get clean && \
```

Comment:
❤️ for indenting this. Make it much clearer that it's a continuation of the line before.